### DISCUSSION TITLE
Improvement: Entries listing: Make entire entry row clickable

### DISCUSSION THREAD

**Discussion by jkorff** [craftcms/cms#8584] [Ideas]
### Description
In the CP entries listing, only the entry title is clickable (without any padding).

It is much better to make the entire row clickable, especially when the entry type is shown which make me look right (which type?) > left (to click), right > left etc.

### Steps to reproduce

1.
2.

### Additional info

- Craft version: 3.0.12
- PHP version:
- Database driver & version:
- Plugins & versions:

**Comment by brandonkelly** [comment]
I’m guessing by “clickable” you mean hyperlinked?

The whole row already is clickable, for selection purposes. And parts of it are double-clickable. (You can double-click on the entry’s status icon to open an inline editor HUD, and if you have a related entry/asset/user/etc. visible on the index, you can double-click on that to edit it as well.

So we would need to be careful not to break the other functionality, and it would need to be very obvious that clicking on the row will take you to the element’s editpage, as opposed to selecting it. Otherwise you could lose other selections as well.

**Comment by jkorff** [comment]
Double-clicking the status icon: I'm discovering all these little goodies I had no idea about! This is not easily discoverable.

I can only select an entry with the checkbox though. Clicking anywhere bar the status icon, link text and the little globe does nothing (Chrome). If you made the cell with "data-titlecell" clickable that would be a great help already.

**Comment by brandonkelly** [comment]
Yeah you’re right. The full row is only selectable within element selection modals.
